_pages/contributing/breaking-changes.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Breaking changes (DRAFT)
+---
+
+## Avoiding breaking changes
+
+Below is a list of things should avoid doing while working on new features and bug fixes. We try to following semantic versioning and doing that means keeping breaking changes for major releases. This list should make you aware of different ways to introduce breaking changes.
+
+- don't change selector of component/directive/pipe
+- don't change or remove exported classes, functions, constants, interfaces and types in `public_api.ts` file
+- don't change or remove interface properties
+- don't change or remove public/protected class members
+- don't change or remove existing public functions parameters (also don't change order)
+- don't add new required properties to interfaces and types (only optional with '?')
```

Comment:
For ease of reading, can we move this line after all function-related ones (so function-related are grouped)?